Little endian configuration for qemu, from linux-mips.org, scheduled for
kernel.org.


diff -Nurd linux-2.6.18.orig/arch/mips/Kconfig linux-2.6.18/arch/mips/Kconfig
--- linux-2.6.18.orig/arch/mips/Kconfig	2006-09-20 05:42:06.000000000 +0200
+++ linux-2.6.18/arch/mips/Kconfig	2006-09-27 23:00:12.124563754 +0200
@@ -557,6 +557,7 @@
 	select SYS_HAS_CPU_MIPS32_R1
 	select SYS_SUPPORTS_32BIT_KERNEL
 	select SYS_SUPPORTS_BIG_ENDIAN
+	select SYS_SUPPORTS_LITTLE_ENDIAN
 	select ARCH_SPARSEMEM_ENABLE
 	help
 	  Qemu is a software emulator which among other architectures also

